README.md

```diff
@@ -270,6 +272,47 @@ To run the tests:
 bundle exec rspec
 ```
 
+### Docker
+
+A Dockerfile is provided in order to interact with the application in development.
+
+Build the docker image:
+
+```sh
+docker-compose build app
+```
+
+Bring up the docker container and it's dependencies:
+
+```sh
+docker-compose up -d
+```
+
+Initialize the database:
+
+```sh
+docker-compose run app bundle exec rails db:reset
+docker-compose run app bundle exec rails db:seed
+```
+
+Interact with the application via localhost:
+```sh
+curl -F 'druid=druid:bj102hs9688' -F 'incoming_version=3' -F 'incoming_size=2070039' -F 'storage_location=spec/fixtures/storage_root01' -F 'checksums_validated=true' http://localhost:3000/v1/catalog
+```
+
+```sh
+curl http://localhost:3000/v1/objects/druid:bj102hs9688
+
+{
+  "id":1,
+  "druid":"bj102hs9688",
+  "current_version":3,
+  "created_at":"2019-12-20T15:04:56.854Z",
+  "updated_at":"2019-12-20T15:04:56.854Z",
+  "preservation_policy_id":1
+}
+```
+
```

Comment:
Should we add a sub-section about pushing the image to DockerHub?

Reply:
Possibly, but I haven't yet as this is mostly for local dev, but I'm more than happy to add that.

Reply:
i'm also OK w/ deferring that till later, but i'm also totally fine to see it added here, whatever's easiest.

Reply:
Yeah, this has potential for prod use, minus the postgres db stuff.